e2e/cypress/tests/integration/messaging/edit_message_spec.js

```diff
@@ -145,4 +145,42 @@ describe('Edit Message', () => {
             cy.get(postText).should('have.text', `${secondMessage} Another new message Edited`);
         });
     });
+
+    it('should discard any changes made after cancelling the edit and opening the edit textbox again should display the original message', () => {
```

Comment:
Can you get the test id from https://github.com/mattermost/mattermost-test-management/pull/83

Reply:
It will be provided in that PR soon.